### PR TITLE
change GO:0060986 to GO:0046879

### LINKED ISSUE #1706: secretory cell and capable of some endocrine hormone secretion
**CL term**
CL_0000163 endocrine cell

**Suggested revision of logical definition**
A proposal has been made to merge GO:0060986 endocrine hormone secretion into GO:0046879 hormone secretion.
Please see https://github.com/geneontology/go-ontology/issues/17084.

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #1706

### CODE REVIEW

**Comment by shawntanzk** [comment]
converting to draft - i think we might need be careful about this

**Comment by shawntanzk** [comment]
I think we need to remove the equiv class (given that it doesn't take into consideration non endocrine hormones) and manually assert as needed. 

FYI @cmungall

**Comment by shawntanzk** [comment]
Decided instead to make endocrine cell equiv:
```
'secretory cell'
 and ('capable of' some 'hormone secretion')
and ('part of' some 'endocrine gland')
```

**Comment by shawntanzk** [comment]
The added and replaced axioms in simple diff seems right to me

Notes on removed axioms:

[adrenocorticotropic hormone secreting cell](http://purl.obolibrary.org/obo/CL_0000467) SubClassOf [endocrine cell](http://purl.obolibrary.org/obo/CL_0000163) <- ACTH is produced in the pituitary, [this site](http://howmed.net/physiology/hormones-non-endocrines/) seems to suggest that it can also be non endocrine. If I were to query for endocrine cells, I probably would expect cells that secrete ACTH though? We could create a new class for specifically endocrine ACTH cells, but I'm not sure if that's ideal. I will search more about lymphocytes secreting ACTH

[gonadotropin releasing neuron](http://purl.obolibrary.org/obo/CL_0011111) SubClassOf [endocrine cell](http://purl.obolibrary.org/obo/CL_0000163)
Should have the parent, as the def states, neuroendocrine cell instead. That should place it back in endocrine cell.

**Comment by bvarner-ebi** [comment]
> Decided instead to make endocrine cell equiv:
> 
> ```
> 'secretory cell'
>  and ('capable of' some 'hormone secretion')
> and ('part of' some 'endocrine gland')
> ```

In the case of, for example, [endocrine cells of the GI tract](https://pubmed.ncbi.nlm.nih.gov/10524284/), is the intestine considered an 'endocrine gland'?

**Comment by shawntanzk** [comment]
actually I'm stupid, theres just endocrine system which should be more encompassing - ill let that run with the diff here and see

**Comment by bvarner-ebi** [comment]
Hi, @shawntanzk, is this still a draft? Once taken out of draft status, I will review.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -4681,8 +4681,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D055098"^^xsd:string) ob
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000163 "FMA:83809"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000163 "endocrinocyte"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000163 "endocrine cell"^^xsd:string)
-EquivalentClasses(obo:CL_0000163 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0060986)))
-SubClassOf(obo:CL_0000163 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002368))
+EquivalentClasses(obo:CL_0000163 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000949) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0046879)))
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:4684]:
```suggestion
EquivalentClasses(obo:CL_0000163 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002368) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0046879)))
```

**Comment by bvarner-ebi** [comment]
@shawntanzk,

> If I were to query for endocrine cells, I probably would expect cells that secrete ACTH though?

If I were to query for endocrine cells, I would expect to find CL:0002309 'corticotroph' (which is currently missing synonym "corticotrope"), but I would not expect to find every cell type that secretes ACTH for the reason you mention.

Regarding CL:0000163 ‘endocrine gland’, I added a suggested change for the EA. 
I also propose the following edit for the text definition:

CURRENT: A cell of an endocrine gland, ductless glands that secrete substances which are released directly into the circulation and which influence metabolism and other body functions.

PROPOSED: A cell that is part of an endocrine gland and is capable of secreting substances directly into the circulation to influence metabolism and other body functions.

**Review by bvarner-ebi — COMMENTED** [review]
See inline comments.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:4684]: lol I changed it to endocrine system from gland due to your comment: 

> In the case of, for example, [endocrine cells of the GI tract](https://pubmed.ncbi.nlm.nih.gov/10524284/), is the intestine considered an 'endocrine gland'?

**Review by dosumis — CHANGES_REQUESTED** [review]
What about cells secreting hormones that at act locally (in a paracrine fashion) within an endocrine organ?  Also this passes the buck to Uberon, who still need a way to define endocrine organs.  I'll post a proposed solution on the main ticket (this should be applicable to Uberon too).

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:4684]: Understandable. Given the complexities, maybe using subclasses instead of an EA is more appropriate for now.

**Comment by bvarner-ebi** [comment]
Hi, @shawntanzk, can this PR be merged and/or closed? 
Given ongoing discussion in the PR, perhaps creating a new ticket is appropriate.

**Comment by shawntanzk** [comment]
> Hi, @shawntanzk, can this PR be merged and/or closed?
Given ongoing discussion in the PR, perhaps creating a new ticket is appropriate.

Have already draft it as I am not sure what the next steps are for this. I don't think there's a need to close if its a draft

**Comment by bvarner-ebi** [comment]
> > Hi, @shawntanzk, can this PR be merged and/or closed?
> > Given ongoing discussion in the PR, perhaps creating a new ticket is appropriate.
> 
> Have already draft it as I am not sure what the next steps are for this. I don't think there's a need to close if its a draft

If there is no clear action within the next week, let's close and move back to an issue so drafts aren't hanging around. 
Trying to get to the point in CL where PRs are made only after a clear action plan is described in a linked issue.

**Comment by shawntanzk** [comment]
> so drafts aren't hanging around

see https://github.com/obophenotype/cell-ontology/pull/1713#issuecomment-1373385992

**Pull request closed without merging**